client-side/src/components/GenericButton.scss

```diff
@@ -0,0 +1,25 @@
+@import url('https://fonts.googleapis.com/css2?family=Nanum+Gothic&display=swap');
+
+.buttonWrapper{
+    .genericButton{
+        font-family:  "Nanum Gothic", sans-serif ;
+        font-optical-sizing: auto ;
+        font-style: normal ;
+        border-radius: 1em ;
+        border: 2px ;
+        border-style: solid ;
+        border-color: rgb(103, 252, 210) ;
+    
+    }
+    
+    .genericButton.primary{
```

Comment:
in scss, you should use hireraci, so you can put the .primary and secondary inside the .genericButton with sing the & sign
will be something like:
.buttonWrapper{
    .genericButton{
      .... all generic button styles
      &.primary {
        ...all primary styles
      }
      &.secondary {
         ...all secondary styles
      }